neurom/__init__.py

```diff
@@ -68,3 +70,5 @@
 AXON = NeuriteType.axon
 SOMA = NeuriteType.soma
 ANY_NEURITE = NeuriteType.all
+
+__version__ = version(__package__)
```

Comment:
I think you should add that at the top before the other imports:
https://peps.python.org/pep-0008/#module-level-dunder-names

Reply:
Ok, done